### ISSUE TITLE
Console spam error

### ISSUE THREAD

**Issue by Yorkforce** [prism/Prism#91]
Currently getting this in the console:
https://pastebin.com/48FjKPde

It repeats the least 5 lines around 1000 times, every time it is triggered, which is alot.
Kinda making the console unusable and the logs very hard to work with.

**Comment by viveleroi** [comment]
I can't tell if this is an issue from Prism or Sponge. Which version of sponge are you using? Vanilla or Forge?

@Meronat Looks like this is related to the sponge task scheduler. Any thoughts?

**Comment by Meronat** [comment]
Yeah I'm not exactly sure what's causing this, but I will look into it next time I'm at a computer. @Yorkforce What versions of Sponge and LuckPerms are you using?

**Comment by Yorkforce** [comment]
@viveleroi @Meronat We're using spongeforge-1.10.2-2254-5.2.0-BETA-2259 with LuckPerms-Sponge-3.0.63, this is on the Invasion v1.0.0 modpack
